system-variables.md

```diff
@@ -175,6 +231,12 @@ mysql> SELECT * FROM t1;
 - Unit: Seconds
 - This variable represents the idle timeout of the interactive user session, which is measured in seconds. Interactive user session refers to the session established by calling [`mysql_real_connect()`](https://dev.mysql.com/doc/c-api/5.7/en/mysql-real-connect.html) API using the `CLIENT_INTERACTIVE` option (for example, MySQL shell client). This variable is fully compatible with MySQL.
 
+### last_insert_id
+
+- Scope: SESSION
+- Default value: ""
+- MySQL Compatibility variable. Not used in TiDB. Use the `LAST_INSERT_ID()` function instead.
```

Comment:
I think we should fix this (it is easy to do with a SetSession func) rather than document it as broken.

Some connectors may require it.

Reply:
Could you open an issue for this? Then we can update the docs once it is fixed.

Reply:
https://github.com/pingcap/tidb/pull/28072